src/Documentation/Action/EntrypointAction.php

```diff
@@ -37,13 +37,13 @@ public function __construct(
         private readonly ResourceNameCollectionFactoryInterface $resourceNameCollectionFactory,
         private readonly ProviderInterface $provider,
         private readonly ProcessorInterface $processor,
-        private readonly array $documentationFormats = []
+        private readonly array $documentationFormats = [],
```

Comment:
This is likely a bug in Symfony variant. AFAIU, `formats` should be injected here, not `docs_formats`.

Reply:
indeed both implementation are not necessary the same

Reply:
Laravel has no differenciation between both formats yet?

Reply:
That would be cool to be able to render SwaggerUI when requesting `/api` as we do for the Symfony variant.